src/components/Secrets/SecretsListView/SecretsListHeader.tsx

```diff
@@ -1,6 +1,7 @@
 export const secretsTableColumnClasses = {
   secretType: 'pf-m-width-30',
   name: 'pf-m-width-30 wrap-column',
+  components: 'pf-m-width-30',
```

Comment:
nit: shouldn't we maybe adjust the width here for each column class to be `-width-20`, since we're adding a new column? :)

Reply:
good points. it is a bit oversize.  :-). let me change it.